invenio/legacy/refextract/config.py

```diff
@@ -17,32 +17,37 @@
 # along with Invenio; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
 
-"""RefExtract configuration"""
+"""refextract configuration."""
+
+
+from invenio.base.config import CFG_VERSION
 
 from __future__ import unicode_literals
```

Comment:
please make use that this is the **first** import in the file just after docstring.